doc/changes.rst

```diff
@@ -81,6 +81,16 @@ Autoremove command
  * Dropped `<spec>` positional argument since the usecase is sufficiently covered by the `remove` command.
  * Specific `autoremove-n`, `autoremove-na`, and `autoremove-nevra` variants of the command are not supported.
 
+Distro-sync command
+-------------------
+ * When any argument does not match any package or package is not installed, DNF5 will fail. The behavior can be
+   modify by the `--skip-unavailable` option.
```

Comment:
Only minor fixes/enhancements (an article and modify -> modified):

When any argument does not match any package or **a** package is not installed, DNF5 will fail. The behavior can be **modified using** the `--skip-unavailable` option.